Content.Shared/RCD/Systems/RCDSystem.cs

```diff
@@ -89,11 +88,15 @@ private void OnRCDSystemMessage(EntityUid uid, RCDComponent component, RCDSystem
         if (!component.AvailablePrototypes.Contains(args.ProtoId))
             return;
 
-        if (!_protoManager.HasIndex(args.ProtoId))
+        RCDPrototype? prototype;
+        if (!_protoManager.Resolve<RCDPrototype>(args.ProtoId, out prototype))
             return;
 
         // Set the current RCD prototype to the one supplied
         component.ProtoId = args.ProtoId;
+
+        _adminLogger.Add(LogType.RCD, LogImpact.Low, $"{ToPrettyString(args.Actor):user} set RCD mode to: {prototype.Mode} : {prototype.Prototype}");
```

Comment:
```suggestion
        _adminLogger.Add(LogType.RCD, LogImpact.Low, $"{args.Actor} set RCD mode to: {prototype.Mode} : {prototype.Prototype}");
```
this is done automatically nowadays